assets/js/modules/reader-revenue-manager/components/dashboard/PublicationApprovedOverlayNotification.stories.js

```diff
@@ -48,6 +50,15 @@ export default {
 	decorators: [
 		( Story, { args } ) => {
 			const setupRegistry = ( registry ) => {
+				provideModules( registry, [
+					{
+						slug: READER_REVENUE_MANAGER_MODULE_SLUG,
+						active: true,
+						connected: true,
+						setupComplete: true,
```

Comment:
`setupComplete` isn't a key for modules, that's essentially what `connected` represents.

```suggestion
```